msal/application.py

```diff
@@ -672,6 +672,7 @@ def _decide_broker(self, allow_broker, enable_pii_log):
                 "allow_broker is deprecated. "
                 "Please use PublicClientApplication(..., "
                 "enable_broker_on_windows=True, "
+                "enable_broker_on_linux=True, "
                 "enable_broker_on_mac=...)",
```

Comment:
**issue:** The `allow_broker=True` is only equivalent to - and can only be translated as - `enable_broker_on_windows=True`, not the other two. Sorry if the previous `enable_broker_on_mac=...` wording causes any confusion. Now I suggest to replace both the linux and mac lines with a `...)`.

```suggestion
                "...)",
```